charts/posthog/README.md

```diff
@@ -524,3 +532,18 @@ Upgrade "posthog" failed: cannot patch "posthog-posthog-redis-master" with kind
 ```
 
 Run the following and `helm upgrade` again: `kubectl delete --namespace NAMESPACE sts posthog-posthog-redis-master`
+  
+### Upgrading from 2.x.x
+  
+3.0.0 changes the way ZK is run in the chart. ZK has been spun out and is now a cluster being used for Kafka and ClickHouse. An unfortunate side effect of that is that Kafka statefulset must be deleted. The reason for this is because Kafka records the cluster ID in ZooKeeper and when you swap it out it complains that the cluster ID has changed and refuses to start.
+  
+The error you will see from Kafka pod while upgrading:
+```
+  [2021-07-23 14:24:27,143] ERROR Fatal error during KafkaServer startup. Prepare to shutdown (kafka.server.KafkaServer)
+kafka.common.InconsistentClusterIdException: The Cluster ID TYP8xsIWRFWkzSYmO_YWEA doesn't match stored clusterId Some(CizxEcefTou4Ehu65rmpuA) in meta.properties. The broker is trying to join the wrong cluster. Configured zookeeper.connect may be wrong.
+  ```
+How to fix?
+- Delete Kafka stateful set `kubectl -n posthog delete sts posthog-posthog-kafka`
+- Delete Kafka pods `kubectl -n posthog delete pod posthog-posthog-kafka-0`
```

Comment:
What happens to the Posthog instance while we run this?

Reply:
it goes down. It's similar to what happens with the fix for going from 1.x.x to 2.x.x